src/libcore/iter.rs

```diff
@@ -148,7 +148,7 @@ pub trait Iterator {
         last
     }
 
-    /// Loops through `n` iterations, returning the `n`th element of the
+    /// Loops through `n` iterations, returning the next (`n+1`th) element of the
```

Comment:
I'd prefer the different version. Perhaps say *skip* and *next element after that*?